RandLAPACK/drivers/rl_cqrrp.hh

```diff
@@ -269,40 +272,36 @@ int CQRRP_blocked<T, RNG>::call(
     T* Work2    = ( T * ) calloc( n, sizeof( T ) );
     //*******************POINTERS TO DATA REQUIRING ADDITIONAL STORAGE END*******************
 
-    T norm_A     = lapack::lange(Norm::Fro, m, n, A, lda);
-    T norm_A_sq  = std::pow(norm_A, 2);
-    T norm_R     = 0.0;
-    T norm_R11   = 0.0;
-    T norm_R12   = 0.0;
-    T norm_R_i   = 0.0;
-    T approx_err = 0.0;
-
     if(this -> timing) {
         preallocation_t_stop  = high_resolution_clock::now();
         preallocation_t_dur   = duration_cast<microseconds>(preallocation_t_stop - preallocation_t_start).count();
         saso_t_start = high_resolution_clock::now();
```

Comment:
Please replace "saso" in sketching-related timing variables with "sketch."

Reply:
Replaced it with skop